From 79135c7585402eadeb9e06cbf3138f0d191b15d9 Mon Sep 17 00:00:00 2001
From: Paul Varner <fuzzyray@gentoo.org>
Date: Wed, 16 May 2012 17:27:54 -0500
Subject: [PATCH 3/3] Change sed statement in euse to fix bug 410365.

The sed statement was looking for the useflag as part of the regular
expression.  Since the input to the statement had already grepped for
the relevant entries from use.local.desc, this is unneccessary.  Changed
the sed statement to look for '(.*)  *' instead of '(${1}) *'.
---
 bin/euse |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/bin/euse b/bin/euse
index 27adb72..4d2c15d 100755
--- a/bin/euse
+++ b/bin/euse
@@ -781,7 +781,7 @@ showdesc() {
 			fi
 			# Fetch all the packages data using this flag
 			infos=$( grep -h ":${1}  *-" ${ALL_PORTDIRS[@]/%//profiles/use.local.desc} 2> /dev/null \
-				| sed -re "s/^([^:]+):(${1}) *- *(.+)/\1|\2|\3/g")
+				| sed -re "s/^([^:]+):(.*)  *- *(.+)/\1|\2|\3/g")
 			OIFS=$IFS; IFS=$'\n'; infos=($infos); IFS=$OIFS;
 			for line in "${infos[@]}"; do
 				OIFS=$IFS; IFS="|"; line=($line); IFS=$OIFS
-- 
1.7.8.6

